### PR TITLE
Hw mobile 2

### PR DESCRIPTION
**Pull request by andrei-mikhailovskii**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,16 +8,32 @@
...
     <dependencies>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>${selenium-java.version}</version>
+        </dependency>
         <dependency>
             <groupId>io.appium</groupId>
             <artifactId>java-client</artifactId>
-            <version>7.3.0</version>
```
Comment (komyagin) [inline pom.xml:15]: Why didn't you move the dependency version to the properties section as you did with the rest?

File: pom.xml [hunk 2]
```diff
@@ -8,16 +8,32 @@
...
+            <version>${java-client.version}</version>
         </dependency>
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>7.1.0</version>
```
Comment (komyagin) [inline pom.xml:20]: Why didn't you move the dependency version to the properties section as you did with the rest?

File: pom.xml [hunk 3]
```diff
@@ -39,11 +55,35 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven-compiler-plugin.version}</version>
                 <configuration>
                     <source>1.8</source>
                     <target>1.8</target>
```
Comment (komyagin) [inline pom.xml]: Could be moved to the properties section

File: pom.xml [hunk 4]
```diff
@@ -69,5 +109,4 @@
         </profile>
     </profiles>
 
-
 </project>
```
Comment (komyagin) [inline pom.xml]: Add a new blank line at the end of the file

File: src/main/java/setup/BaseTest.java [hunk 5]
```diff
@@ -1,52 +1,78 @@
...
     }
 
-    @Parameters({"platformName","appType","deviceName","browserName","app"})
+    @Parameters({"platformName", "appType", "deviceName", "browserName", "app"})
     @BeforeSuite(alwaysRun = true)
-    public void setUp(String platformName, String appType, String deviceName, @Optional("") String browserName, @Optional("") String app) throws Exception {
-        System.out.println("Before: app type - "+appType);
+    public void setUp(String platformName, String appType, String deviceName,
+                      @Optional("") String browserName, @Optional("") String app) throws Exception {
+        System.out.println("Before: app type - " + appType);
```
Comment (komyagin) [inline src/main/java/setup/BaseTest.java]: System.out.println? Why do you need it here?

File: src/main/java/setup/BaseTest.java [hunk 6]
```diff
@@ -1,52 +1,78 @@
...
     @AfterSuite(alwaysRun = true)
-    public void tearDown() throws Exception {
-        System.out.println("After");
-        appiumDriver.closeApp();
+    public void tearDown() {
+        try {
+            System.out.println("After");
+            appiumDriver.closeApp();
+        } catch (NullPointerException nullPointerException) {
+            System.err.println("Appium driver is null: " + nullPointerException);
```
Comment (komyagin) [inline src/main/java/setup/BaseTest.java]: If you want to see error messages use logger

File: src/test/java/scenarios/NativeMobileTests.java [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+    }
+
+    /*@Test(groups = {"native"}, description = "This simple test just click on the Sign In button")
+    public void simpleNativeTest() throws IllegalAccessException, NoSuchFieldException, InstantiationException {
+        getPo().getWelement("signInBtn").click();
+        System.out.println("Simplest Android native test done");
+
+    }*/
```
Comment (komyagin) [inline src/test/java/scenarios/NativeMobileTests.java]: Why did you leave commented code in the PR?

File: src/test/java/scenarios/WebMobileTests.java [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+                wd -> ((JavascriptExecutor) wd).executeScript("return document.readyState").equals("complete")
+        );
+
+        // Check IANA homepage title
+        assert ((WebDriver) getDriver())
+        .getTitle().equals("Internet Assigned Numbers Authority") : "This is not IANA homepage";
+
+        // Log that test finished
+        System.out.println("Site opening done");
+    }*/
```
Comment (komyagin) [inline src/test/java/scenarios/WebMobileTests.java]: Why did you leave commented code in the PR?

**Review by komyagin — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (andrei-mikhailovskii) [reply pom.xml:15]: moved, thank you

File: pom.xml (hunk 2 above)
Comment (andrei-mikhailovskii) [reply pom.xml:20]: moved

File: pom.xml (hunk 3 above)
Comment (andrei-mikhailovskii) [reply pom.xml]: moved

File: pom.xml (hunk 4 above)
Comment (andrei-mikhailovskii) [reply pom.xml]: added

**Review by komyagin — APPROVED** [review]

File: src/main/java/setup/BaseTest.java (hunk 5 above)
Comment (andrei-mikhailovskii) [reply src/main/java/setup/BaseTest.java]: That was in the original code, I left it just to see what's going on in the console, but now I removed it, thank you

File: src/main/java/setup/BaseTest.java (hunk 6 above)
Comment (andrei-mikhailovskii) [reply src/main/java/setup/BaseTest.java]: Replaced with logger

File: src/test/java/scenarios/NativeMobileTests.java (hunk 7 above)
Comment (andrei-mikhailovskii) [reply src/test/java/scenarios/NativeMobileTests.java]: That was the original code by Maksim. Removed

File: src/test/java/scenarios/WebMobileTests.java (hunk 8 above)
Comment (andrei-mikhailovskii) [reply src/test/java/scenarios/WebMobileTests.java]: That was the original code by Maksim. Removed

**Pull request merged**